src/components/structures/HomePage.tsx

```diff
@@ -109,16 +109,18 @@ const HomePage: React.FC<IProps> = ({ justRegistered = false }) => {
         const brandingConfig = config.branding;
         let logoUrl = "themes/element/img/logos/element-logo.svg";
         let logoUrlSecondary = SettingsStore.getValue(UIFeature.SecondaryLogoUrl);
+        let logoSecondaryDescription = config['secondary_logo_description'];
         if (brandingConfig && brandingConfig.authHeaderLogoUrl) {
             logoUrl = brandingConfig.authHeaderLogoUrl;
         }
-       const logoUrlSecondaryStyle = {
-           margin: '0 10px'
+       const logoSecondaryStyle = {
+           margin: '0 10px',
+           height: '55px'
```

Comment:
May I know for my curiosity why would we want only height to be controlled from config not other styles  ? Second question would be since secondary logo is not in matrix this value wont apply if secondary logo is not present on UI so I am bit confused about height needing to be configurable ? Please enlighten me .

Reply:
This suggestion has been implemented. This took me 2hrs now and I have implemented in a way that will make everything related to secondaryLogo configurable such as img location, margin, height, description of img in single spot. UIFeature.secondaryLogoUrl has been removed as new config is going to centralise everything now on for secondary image. I am happier than before now. Thanks for reminder as with this change I have centralised all values needed into one config value.